package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "leader-board",
+  "version": "1.0.0",
+  "description": "",
+  "private": "true",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack --watch"
```

Comment:
- [OPTIONAL] I would suggest you add the `npm start` command script to your `package.json` file to help you start your application without the need of building your application.
```
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Reply:
Thanks @leonardodiasb for the suggestion